README.md

```diff
@@ -81,6 +82,14 @@
 
 ## 📦 Installation
 
+### Nix
```

Comment:
The `README` is automatically generated from the documentation in `docs/` so this change needs to appear there as well.